### ISSUE TITLE
ClientPlayer -> MainClientPlayer, AbstractClientPlayer -> ClientPlayer

### ISSUE THREAD

**Issue by Runemoro** [asiekierka/MinecraftPOMF#597]
I think ClientPlayer should be named MainClientPlayer to make it clear that it's only being used for the main (first person) player entity (the others are OtherClientPlayers). Then we could rename AbstractClientPlayer to ClientPlayer.

**Comment by Runemoro** [comment]
@kashike Why do you disagree? Other players are also client players, so it would be confusing for `OtherClientPlayer` to not extend `ClientPlayer`.

**Comment by liach** [comment]
Or should we name `ClientPlayer` to `ControlledPlayer` as the player is controlled by the client (also rename the `isMainPlayer` method)

**Comment by kashike** [comment]
I disagree with the renaming of `ClientPlayer` and `AbstractClientPlayer`. `OtherClientPlayer` could use a rename.

**Comment by Runemoro** [comment]
But `OtherClientPlayer` (or whatever it's renamed to) *should* extend `ClientPlayer`, it is a client-side player entity, right?
